features/step_definitions/division_steps.rb

```diff
@@ -0,0 +1,55 @@
+require 'calculator'
+
+When /^I divide two numbers$/ do
+	@quotient = Calculator.new.divide 4,2
+end
+
+Then /^I get the quotient$/ do
+	expect(@quotient).to eq 2
+end
+
+When /^I divide a number by zero$/ do
+	begin
+	Calculator.new.divide 4,0
+	rescue Exception => e
+	end
+end
+
+Then /^I receive an error$/ do
+	#catch 
+	expect{Calculator.new.divide 4,0}.to raise_exception(ZeroDivisionError)

```

Comment:
This effectively tests the scenario despite your `When` step. No points were taken off. :+1: